src/mapblock_mesh.h

```diff
@@ -165,7 +168,7 @@ struct PreMeshBuffer
 {
 	TileSpec tile;
 	std::vector<u16> indices;
-	std::vector<video::S3DVertex> vertices;
+	std::vector<video::S3DVertexTangents> vertices;
```

Comment:
You can get around the S3DVertex/S3DVertexTangents problem by either defining two versions of PreMeshBuffer, or defining PreMeshBuffer as a template struct